extensions/kyma/kyma.yaml

```diff
@@ -95,13 +95,32 @@ data:
       metadata.creationTimestamp: Created at
       alert.beta: "CAUTION: The Service Level Agreements (SLA) and Support obligations do not apply to Beta modules and functionalities. If Beta modules or functionalities directly or indirectly affect other modules, the Service Level Agreements (SLA) and Support for these modules are limited to priority levels P3 (Medium) or P4 (Low). Thus, Beta releases are not intended for use in customer production environments."
   injections: |-
-    - name: Kyma
+    - name: Kyma Modules
       widget: Panel
       source: $
+      visibility: $not($exists($item.spec.modules.channel))
       targets:
         - slot: top
           location: ClusterOverview
       order: 1 
+      children: 
+        - widget: MessagePanel
+          source: $
+          name: Test panel
+          title: Seems that you don't have any Kyma Modules Configured
+          subtitle: Add at least one Kyma Module. Add your module under "kyma-system -> Kyma"
+          children:
+            - widget: Wizard
+              name: Add Module
+              wizard: module-wizard
+              visibility: $not($exists($item.spec.modules.channel))
```

Comment:
We show this button even if `module-wizard` is not present. Moreover we have it as default for kymas but we don't provide wizard as well.
Either hide when `module-wizard` is not on cluster or provide `module-wizard` as default wizard